doc/CMakeLists.txt

```diff
@@ -91,6 +91,12 @@ add_custom_target(
     ${DOXYGEN_EXECUTABLE} ${DOXYFILE_OUT}
 )
 
+set_target_properties(
```

Comment:
Nothing wrong here, but this commit message https://github.com/zephyrproject-rtos/zephyr/pull/36341/commits/041b0aaff76cf1c43e022bd11aeef942caf92d43 says:

> Indicate which files have to be cleaned for each target. CMake version
> requirement has been upgraded to 3.15 in order to support this feature.


maybe adjust that message to:
> Indicate which files have to be cleaned for each target.
>
> This is only supported for CMake >=3.15 and is ignored for older CMake versions.
> Build will still succeed but `<build-cmd> clean` will not clean the additional build files.

Reply:
adjusted commit message, thx